<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SqlWin</class>
 <widget class="QMainWindow" name="SqlWin">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>627</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab_food">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>菜品</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout" columnstretch="9,0">
        <item row="0" column="0">
         <widget class="QLineEdit" name="lineEdit_search">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="placeholderText">
           <string>搜索</string>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pBtn_foodSwap">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>交换</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pBtn_foodUp">
            <property name="text">
             <string>上移</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pBtn_foodDown">
            <property name="text">
             <string>下移</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pBtn_foodDel">
            <property name="text">
             <string>删除</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pBtn_foodAdd">
            <property name="text">
             <string>添加</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>菜名</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>品类</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QVBoxLayout" name="vBoxLayout_food">
          <property name="spacing">
           <number>0</number>
          </property>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_order">
       <attribute name="title">
        <string>订单</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3" columnstretch="9,1">
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,9">
            <item>
             <widget class="QPushButton" name="pBtn_orderBack">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(85, 255, 255);</string>
              </property>
              <property name="inputMethodHints">
               <set>Qt::InputMethodHint::ImhNone</set>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset theme="QIcon::ThemeIcon::EditUndo"/>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_account">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(85, 255, 255);</string>
            </property>
            <property name="text">
             <string>隐藏账号</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pBtn_orderSwap">
            <property name="text">
             <string>导出</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pBtn_orderClear">
            <property name="text">
             <string>清空</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="vBoxLayout_order">
          <item>
           <widget class="QStackedWidget" name="stackedWidget">
            <property name="currentIndex">
             <number>1</number>
            </property>
            <widget class="QWidget" name="page"/>
            <widget class="QWidget" name="page_2"/>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_tcp">
       <attribute name="title">
        <string>运营</string>
       </attribute>
       <widget class="QWidget" name="widget" native="true">
        <property name="geometry">
         <rect>
          <x>12</x>
          <y>12</y>
          <width>781</width>
          <height>551</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>30</y>
           <width>591</width>
           <height>34</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,10,1">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>许可证</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_passwd">
            <property name="text">
             <string>I am passwd</string>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::EchoMode::Password</enum>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pBtn_tcpEnter">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset theme="QIcon::ThemeIcon::UserAvailable"/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QTextBrowser" name="textBrowser_tcp">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>120</y>
           <width>591</width>
           <height>381</height>
          </rect>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>70</y>
           <width>591</width>
           <height>34</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,15,0">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>port</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_tcpPort">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>8888</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pBtn_tcpOpen">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>营业</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>230</y>
           <width>160</width>
           <height>87</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_serverTitle">
            <property name="font">
             <font>
              <pointsize>26</pointsize>
             </font>
            </property>
            <property name="text">
             <string>服务器</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_serverSta">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>关机中...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
